clang/test/CIR/CodeGen/vtable-thunk-compare-codegen.cpp

```diff
@@ -0,0 +1,35 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.cir.ll
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -emit-llvm %s -o %t.codegen.ll
+// RUN: FileCheck --input-file=%t.cir.ll %s --check-prefix=CIR
```

Comment:
I think LLVM vs OCCG here would be more clear. Why there's no proper CIR output here?